Source/Fuse.Elements/Caching/ElementBatch.uno

```diff
@@ -140,12 +140,28 @@ namespace Fuse.Elements
 				elm.AbsoluteZoom)), CachingRectPadding);
 		}
 
+		Box GetRenderBounds(Element elm)
+		{
+			var t = elm.InternLocalTransformInternal;
+			return VisualBounds.BoxTransform((Box)elm.LocalRenderBounds, t);
```

Comment:
This seems to break the Infinite and Empty bounds, both of which are important at times.
A transformed 0,0,0 is not the same as empty -- it will result in bounds that are too big in many situations. Empty needs to be empty.